python/guide/packages/gluon/index.rst

```diff
@@ -125,6 +131,17 @@ Advanced Topics
 
       Speed up training with hybrid networks.
 
+   .. card::
+      :title: Custom Operators
+      :link: customop.html
+
+      Building custom operators with numpy.
+ 
+   .. card::
```

Comment:
I'd put this up in the Data section.

Reply:
Moved into the Data section

Reply:
Moved to the data section

Reply:
I was referring to line 140 and the card contents below it. Custom op was fine...

Reply:
Interesting.... the preview makes i look like I was flagging "Building customer operators with numpy." But I wasn't. I put the comment on line 140... next time I'll be more specific and not use "this"❗️